src/documents/IoCTestingFramework.md

```diff
@@ -122,151 +116,398 @@ fluid.tests.globalCatTest = function (catt) {
+QUnit [modules](https://api.qunitjs.com/QUnit.module/), and within that an entry named `tests`, holding an array of structures corresponding to QUnit [test](https://api.qunitjs.com/QUnit.test/). In 
+ordinary use, each element `tests` then contains a member named `sequence` holding a list of [*fixture records*](#supported-fixture-records).
 
-For highly dynamic tests, you may want to assemble your sequence elements dynamically for each test rather than list them statically. In this case,
-you can replace `modules` with the entry `moduleSource` which takes the same form as an [invoker](Invokers.md] record with entries `func`/`funcName` and `args`,
-and returns a list of fixtures in the same format as `modules`. Again, in a real example, this would use the `sequence` form of fixtures shown at the bottom of the page.
+In the very [simple example](#simple-example) above, the member `sequence` is omitted since there is just one fixture record. 
+This record executes a global function, `fluid.tests.globalCatTest` which makes one jqUnit assertion.
+In a realistic use case, each entry within `tests` will instead hold an entry named `sequence` which holds an array of fixture records
+representing sequence points to be attained by the test case - as seen in the [asyncTester example](#-testcaseholder-demonstrating-sequence-record) below. 
 
-```javascript
-// Example of the above fixture written with "moduleSource"
-// Definition of modules as namespaced global so that it is available for others and processing
-fluid.tests.catTesterModules = [{
-    name: "Cat test case",
-    tests: [{
-        expect: 1,
-        name: "Test Global Meow",
-        type: "test",
-        func: "fluid.tests.globalCatTest",
-        args: "{cat}"
-    }
-];
+As well as containing a flat list of fixture records, `sequence` may also contain nested arrays of such records. These nested arrays will be
+flattened into a single array by use of the utility [`fluid.flatten`](CoreAPI.md#fluid-flatten-array-) before being processed. This helps in
+assembling complex sequences out of previously canned sequence segments. However, building up complex, reusable test sequences is best done by
+use of the [`sequenceGrade`](/IoCTestingFramework.md#using-sequencegrade-to-build-up-complex-reusable-test-sequences) element, instead of the `sequence` element.
 
-// A helper function which just returns the global -
-// realistically, it would assemble a sequence using more complex logic
-fluid.tests.getCatModules = function () {
-    return fluid.tests.catTesterModules;
-};
-
-fluid.defaults("fluid.tests.catTester", {
-    gradeNames: ["fluid.test.testCaseHolder"],
-    moduleSource: {
-        funcName: "fluid.tests.getCatModules"
-    }
-});
-```
+An example of the `sequence` element appears below in the [asyncTester example](#-testcaseholder-demonstrating-sequence-record) below.
```

Comment:
Reading through to this point in the documentation as a relative newbie to the IoC testing framework, I was thinking it would be helpful if we showed a relatively simple example of what a test with a <code>sequence</code> record looks like right here, prior to the section on Supported Fixture Records. I found myself scrolling way down the page to find the example that is referenced, and then realizing that it was probably too complex for this stage in the documentation and going back to read more, but feeling unsatisfied. For each new concept, I like to see an example to go with it. Having periodic, progressive (in terms of complexity) examples helps a lot in reinforcing comprehension. You look at an example for something you just read about and can say to yourself "oh, yes, I get it."